src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/SyntaxTriviaListExtensions.vb

```diff
@@ -8,6 +8,15 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
     Friend Module SyntaxTriviaListExtensions
 
+        <Extension()>
+        Public Function ContainsCommentOrLineContinue(list As IEnumerable(Of SyntaxTrivia)) As Boolean
```

Comment:
ContainsCommentOrLineContinuation

Reply:
It is complex but the rules are complex, if you want to keep the comments and _ formatting. I newer version of NormalizeWhitespace or the Formatter that @sharwell was working on for VB would be better but even larger and more complex. I would tackle that if there is interest. I have a similar version for removing ByVal and keeping trivia but it is even more complex. Just keeping the comments would be easier but you would end up with ' Comment 1 ' Comment 2...EOL with the ' optional. Would you prefer that?